adapters/mqtt-vertx/src/main/java/org/eclipse/hono/adapter/mqtt/VertxBasedMqttProtocolAdapter.java

```diff
@@ -264,6 +267,8 @@ private void handleEndpointConnectionWithAuthentication(final MqttEndpoint endpo
         if (endpoint.auth() == null) {
             LOG.trace("no auth information in endpoint found");
```

Comment:
This could go away (logging below is better IMHO)

Reply:
I think that it's useful because at a deeper log level you can know what's the reason for refusing. No wrong provided username and/or password but just auth info not provided.

Reply:
Yes, you are right. Better keep it +1